thirdparty/libjpeg-turbo/patches/0001-cmake-generated-headers.patch

```diff
@@ -175,400 +175,42 @@ index 0000000000..f6171bf846
 +
 +#endif
 diff --git a/thirdparty/libjpeg-turbo/src/jmorecfg.h b/thirdparty/libjpeg-turbo/src/jmorecfg.h
-new file mode 100644
-index 0000000000..f7de737edb
---- /dev/null
+index 89c7842c87..f7de737edb 100644
+--- a/thirdparty/libjpeg-turbo/src/jmorecfg.h
 +++ b/thirdparty/libjpeg-turbo/src/jmorecfg.h
-@@ -0,0 +1,389 @@
-+// Originally generated by libjpeg-turbo's cmake build, then modified to support multiple platforms.
-+
-+/*
-+ * jmorecfg.h
-+ *
-+ * This file was part of the Independent JPEG Group's software:
-+ * Copyright (C) 1991-1997, Thomas G. Lane.
-+ * Modified 1997-2009 by Guido Vollbeding.
-+ * Lossless JPEG Modifications:
-+ * Copyright (C) 1999, Ken Murchison.
-+ * libjpeg-turbo Modifications:
-+ * Copyright (C) 2009, 2011, 2014-2015, 2018, 2020, 2022, D. R. Commander.
+@@ -1,3 +1,5 @@
++// Modified to remove lossless jpeg support.
++
+ /*
+  * jmorecfg.h
+  *
+@@ -8,6 +10,8 @@
+  * Copyright (C) 1999, Ken Murchison.
+  * libjpeg-turbo Modifications:
+  * Copyright (C) 2009, 2011, 2014-2015, 2018, 2020, 2022, D. R. Commander.
 + * Godot modifications:
 + * Copyright (c) 2014-present Godot Engine contributors (see AUTHORS.md).
```

Comment:
I don't think we should assert copyright for the small config tweaks we're making to the library.